pages/[lang]/index.js

```diff
@@ -1,20 +1,22 @@
 import React, { Component, Fragment } from "react";
 import PropTypes from "prop-types";
-import { observer, inject } from "mobx-react";
+import inject from "hocs/inject";
 import Helmet from "react-helmet";
 import withCatalogItems from "containers/catalog/withCatalogItems";
 import ProductGrid from "components/ProductGrid";
-import trackProductListViewed from "lib/tracking/trackProductListViewed";
+import Layout from "components/Layout";
+// import trackProductListViewed from "lib/tracking/trackProductListViewed";
```

Comment:
Remove commented-out code if it’s not going to be used in the future.

Reply:
Removed in https://github.com/reactioncommerce/example-storefront/pull/667/commits/f99f9adf797a20148632641ef9413b915e711c56